File: dataPreprocessing_v2/PSWF.py
from enum import Enum


class PSWF(Enum):
    PARKEN = "Parken"
    STDFKT = "Standfunktionen"
    WOHNEN = "Wohnen"
    PAD = "PruefenAnDi"
    FAHREN = "Fahren"
    INVALID = "Invalid"


class PSWFStateChange:
    def __init__(self):
        self.gps_coord = None
        self.old_pswf = None
        self.new_pswf = None

File: 2023.04.28_asc2kml/src/asc2kml/PSWF.java
/*
 * Click nbfs://nbhost/SystemFileSystem/Templates/Licenses/license-default.txt to change this license
 * Click nbfs://nbhost/SystemFileSystem/Templates/Classes/Class.java to edit this template
 */
package asc2kml;

/**
 *
 * @author Q262579
 */
public enum PSWF {
    PARKEN("Parken"),
    STDFKT("Standfunktionen"),        
    WOHNEN("Wohnen"),
    PAD("PruefenAnDi"),
    FAHREN("Fahren"),
    INVALID("Invalid");

    public final String label;

    PSWF(String label) {
        this.label = label;
    }
}
